Dockerfile

```diff
@@ -22,12 +34,19 @@ COPY pkg/ pkg/
 ARG VERSION=devel
 ARG TARGETARCH
 ARG TARGETOS
-RUN --mount=type=cache,target=/root/.cache/go-build CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "-X github.com/piraeusdatastore/drbd-shutdown-guard/pkg/vars.Version=$VERSION" -o drbd-shutdown-guard main.go
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    CGO_ENABLED=0 \
+    GOOS=$TARGETOS \
+    GOARCH=$TARGETARCH \
+    go build -a -ldflags "-X github.com/piraeusdatastore/drbd-shutdown-guard/pkg/vars.Version=$VERSION" -o drbd-shutdown-guard main.go
 
-FROM --platform=$TARGETPLATFORM registry.access.redhat.com/ubi9/ubi:latest
+# === Stage 3: Final minimal runtime image ===
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 
+# Copy only the built binaries from previous stages
 COPY --from=utils-builder /drbdsetup /usr/local/sbin/drbdsetup
 COPY --from=go-builder /work/drbd-shutdown-guard /usr/local/sbin/drbd-shutdown-guard
 
+# Define the binary to run
```

Comment:
Remove. No need to have a comment restart what is obviously happening in the two lines below.